Makefile

```diff
@@ -71,6 +76,7 @@ ifeq ("Darwin", "$(shell uname -s)")
 else
 	cd viam-cartographer/scripts && ./setup_cartographer_linux.sh
 endif
+	@make artifact-pull
```

Comment:
[opt] It might be better to only run `make artifact-pull ` during `make test` (line 111). Potentially could also split off a new `make test-setup` if desired

Reply:
We don't want to run it every time we run tests, otherwise it will slow down tests (which are run frequently during development) Rather than adding new commands the entire slam team needs to remember to run whenever they reinstall the system I decided to slightly slow down `make setup` (which should be run infrequently on local machines) in order to ensure that tests would "just work" for the entire team after this change & not require the entire team to remember to run a new make command.